doc/reference/devices_gpu.md

```diff
@@ -53,6 +53,32 @@ Add a specific GPU from the host system as a `physical` GPU device to an instanc
 
 See {ref}`instances-configure-devices` for more information.
 
+### Configuration examples with the Container Device Interface (CDI) mode
+
+Add a specific GPU from the host system as a `physical` GPU device to an instance using the [Container Device Interface](https://github.com/cncf-tags/container-device-interface) (CDI) notation through a _fully-qualified CDI name_:
+
+    lxc config device add <instance_name> <device_name> gpu gputype=physical id=<fully_qualified_CDI_name>
+
+For example, here is how you can add the first available NVIDIA discrete GPU on your system:
+
+    lxc config device add <instance_name> <device_name> gpu gputype=physical id=nvidia.com/gpu=0
+
+For example, here is how you can add the first available NVIDIA discrete GPU on your system:
+
+    lxc config device add c1 gpu0 gpu gputype=physical id=nvidia.com/gpu=0
```

Comment:
This is a duplicate from above, isn't it? Just with `c1` and `gpu0`. I would stick to `<instance_name>` and `<device_name>` for all examples (since this is what we're doing for the other examples on this page).